tutorials/babble/drink/image_objects.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
```

Comment:
Why are we reintroducing Image_Candidate and BBox objects. Do the ones in context.py that we use for the bike task not work for the drink task?

Reply:
they should, i just copied most of the files that existed in the bike/ folder so we shouldn't include this second file